tools/wdio/plugins/custom-commands.mjs

```diff
@@ -96,18 +96,21 @@ export default class CustomCommands {
      * Sets a permanent scheduled reply for the rum call to include the session
      * replay flag with a value of 1 enabling the feature.
      */
-    browser.addCommand('enableSessionReplay', async function () {
+    browser.addCommand('enableSessionReplay', async function (sampling_rate = 100, error_sampling_rate = 100) {
+      const stMode = Math.random() * 100 < sampling_rate ? 1 : Math.random() * 100 < error_sampling_rate ? 2 : 0
       await browser.testHandle.scheduleReply('bamServer', {
         test: testRumRequest,
         permanent: true,
         body: JSON.stringify({
           stn: 1,
+          sts: 1,
```

Comment:
Assuming bcs does it's stuff correctly, is sts supposed to be uncorrelated to srs? This wouldn't have simply been the case with our side logic with how much conditional branching was happening. I just want to make sure our test server & tests capture real behavior.

Reply:
they will be inherently correlated, but for our purposes we are to treat them independently